lightning/src/ln/onion_payment.rs

```diff
@@ -273,7 +273,37 @@ pub(super) fn create_recv_pending_htlc_info(
 			 intro_node_blinding_point.is_none(), true, invoice_request)
 		}
 		#[cfg(trampoline)]
-		onion_utils::Hop::TrampolineReceive { .. } | onion_utils::Hop::TrampolineBlindedReceive { .. } => todo!(),
+		onion_utils::Hop::TrampolineReceive {
+			trampoline_hop_data: msgs::InboundOnionReceivePayload {
+				payment_data, keysend_preimage, custom_tlvs, sender_intended_htlc_amt_msat,
+				cltv_expiry_height, payment_metadata, ..
+			}, ..
+		} =>
+			(payment_data, keysend_preimage, custom_tlvs, sender_intended_htlc_amt_msat,
+				cltv_expiry_height, payment_metadata, None, false, keysend_preimage.is_none(), None),
+		#[cfg(trampoline)]
+		onion_utils::Hop::TrampolineBlindedReceive {
```

Comment:
I tried to understand the exact difference bewteen `Hop::TrampolineBlindedReceive` and `Hop::TrampolineReceive`, but the enum definition gives the same explanation for both. I am surely missing context, but perhaps something can be added there? (pre-existing ofc)